From 436543acab9441ebb121da74b6433eae43bc8786 Mon Sep 17 00:00:00 2001
From: Pooja Prajod <a0132412@ti.com>
Date: Fri, 24 Mar 2017 18:24:15 +0530
Subject: [PATCH] HACK: playbin: Change default value of flags

The deinterlace flag in playbin is causing
high cpu load which in turn lowers fps of the playback.
The behavior is not yet rootcaused. This is a workaround.

Signed-off-by: Pooja Prajod <a0132412@ti.com>
---
 gst/playback/gstplaybin2.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gst/playback/gstplaybin2.c b/gst/playback/gstplaybin2.c
index ad63600..13439e3 100644
--- a/gst/playback/gstplaybin2.c
+++ b/gst/playback/gstplaybin2.c
@@ -524,7 +524,7 @@ struct _GstPlayBinClass
 #define DEFAULT_SUBURI            NULL
 #define DEFAULT_SOURCE            NULL
 #define DEFAULT_FLAGS             GST_PLAY_FLAG_AUDIO | GST_PLAY_FLAG_VIDEO | GST_PLAY_FLAG_TEXT | \
-                                  GST_PLAY_FLAG_SOFT_VOLUME | GST_PLAY_FLAG_DEINTERLACE | \
+                                  GST_PLAY_FLAG_SOFT_VOLUME | \
                                   GST_PLAY_FLAG_SOFT_COLORBALANCE
 #define DEFAULT_N_VIDEO           0
 #define DEFAULT_CURRENT_VIDEO     -1
-- 
1.9.1

